shaders/circle.fragment.glsl

```diff
@@ -10,4 +10,8 @@ varying lowp float v_antialiasblur;
 void main() {
     float t = smoothstep(1.0 - max(u_blur, v_antialiasblur), 1.0, length(v_extrude));
     gl_FragColor = v_color * (1.0 - t) * u_opacity;
+
+#ifdef OVERDRAW_INSPECTOR
+    gl_FragColor = vec4(1.0);
+#endif
```

Comment:
> changing the color and opacity passed in the draw_* methods

This isn't easily doable for programs that read from textures

> putting the other set in a #ifndef block

Yes, this could be done, but I'd prefer not to. Functionally, both are the same. Using #ifndef would make the overdraw code more intertwined with the rest of the shader and not have any benefit.

I'll leave these lines as they are